index.html

```diff
@@ -11,17 +11,18 @@
     <title>Microverse Porfolio</title>
 </head>
 <body>
-    <div class="nav">
+    <nav class="nav">
         <div class="navigation">
-            <div class="logo">Didier Ganthier</div>
+            <a class="logo" href="#">Didier Ganthier</a>
             <ul class="links">
                 <li><a href="#">Porfolio</a></li>
                 <li><a href="#">About</a></li>
                 <li><a href="#">Contact</a></li>
             </ul>
+            <img class="hamburger" src="images/hamburger.png" alt="">
         </div>  
-    </div>
-    <div class="header">
+    </nav>
+    <section class="header">
         <div class="headline">I'm Didier<br/>Glad to see you!</div>
```

Comment:
- [ ] [OPTIONAL] I am marking this as optional because the previous reviewer omitted it. Kindly replace this `div` with an `h1` tag for semantic and SEO optimization reasons.